src/runtime_src/core/pcie/windows/device_windows.cpp

```diff
@@ -76,19 +76,18 @@ struct ready
 
 struct xmc_sc_presence
 {
-	using result_type = bool;
-
-	static result_type
-		user(const xrt_core::device* device, key_type key)
-	{
-		return true;
-	}
-
-	static result_type
-		mgmt(const xrt_core::device* device, key_type key)
-	{
-		return true;
-	}
+  using result_type = bool;
+
+  static result_type
+    user(const xrt_core::device* device, key_type key)
+  {
+  	return true;
+  }
+  static result_type
+    mgmt(const xrt_core::device* device, key_type key)
```

Comment:
Indention isn't consistent with the the rest of the source code, please remove the indention.